vyper/venom/ir_node_to_venom.py

```diff
@@ -98,22 +98,30 @@ def _findIRnode(ir: IRnode, value: str) -> Optional[IRnode]:
     return None
 
 
-def convert_ir_basicblock(ir: IRnode) -> tuple[IRFunction, IRFunction]:
+def convert_ir_basicblock(ir: IRnode) -> tuple[Optional[IRFunction], Optional[IRFunction]]:
+    runtime_ir = ir
     deploy_ir = _findIRnode(ir, "deploy")
-    assert deploy_ir is not None
```

Comment:
hmm. we should just have a separate function if we want to pass in handrolled IR, as this can never happen from the regular CompilerData pipeline.